modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/WorkflowUser.testcase

```diff
@@ -62,11 +62,46 @@ definition {
 	}
 
 	@description = "This is a test for LRQA-69709. Verify that the user can see the actual state of the user in the workflow process before assigning to itself. "
-	@ignore = "Test Stub"
 	@priority = "3"
 	test CanSeeStateBeforeAssign {
+		Navigator.openURL();
+
+		WorkflowUser.disableCaptchaToCreateNewUser();
+
+		ApplicationsMenu.gotoPortlet(
+			category = "Workflow",
+			panel = "Applications",
+			portlet = "Process Builder");
+
+		Navigator.gotoNavItem(navItem = "Configuration");
+
+		Workflow.configureWorkflow(
+			workflowDefinition = "Single Approver",
+			workflowResourceValue = "User");
+
+		User.logoutPG();
+
+		WorkflowAsset.createNewUserAccount(
+			newPassword = "password",
+			userEmailAddress = "user@liferay.com",
+			userFirstName = "userfn",
+			userLastName = "userln",
+			userScreenName = "usersn");
+
+		User.loginUserPG(
+			password = "test",
+			userEmailAddress = "test@liferay.com");
+
+		UserBar.gotoDropdownItem(dropdownItem = "My Workflow Tasks");
+
+		Navigator.gotoNavTab(navTab = "Assigned to My Roles");
+
+		Workflow.gotoAssetViaTableTitle(workflowAssetTitle = "userfn userln");
 
-		// TODO LRQA-69709 CanSeeStateBeforeAssign pending implementation
+		AssertElementPresent(
+			key_state = "Review",
+			locator1 = "AssetWorkflow#ASSET_STATE");
```

Comment:
Look for a macro for that, maybe Workflow.viewTaskAssignedToMyRoles make this. Thanks!